src/reactive_system_urc/include/ReactiveSystemPath.h

```diff
@@ -0,0 +1,107 @@
+/**
+ * Created by William Gu on Nov 3 2018
+ * Declarations for Reactive System Pathfinder
+ * This node is responsible for moving the robot towards a given GPS goal while
+ * avoiding risky areas
+ */
+
+#ifndef PROJECT_ReactiveSystemTwist_H
+#define PROJECT_ReactiveSystemTwist_H
+
+#include <geometry_msgs/Point32.h>
+#include <geometry_msgs/Pose.h>
+#include <geometry_msgs/Twist.h>
+#include <iostream>
+#include <mapping_msgs_urc/RiskArea.h>
+#include <mapping_msgs_urc/RiskAreaArray.h>
+#include <mapping_msgs_urc/RiskAreaStamped.h>
+#include <math.h>
+#include <nav_msgs/Path.h>
+#include <ros/ros.h>
+#include <sb_geom_msgs/Point2D.h>
+#include <sensor_msgs/LaserScan.h>
+#include <std_msgs/Float64.h>
+
+#define MAX_RISK 100
```

Comment:
Where is this value from? Generally when I think of needing a range like this it would be `[0,1]`?

Reply:
I think Robyn said that the risk values would range from 0 to 100

Reply:
@RobynCastro?

Reply:
The values should range from 0 to 10 right now, but I can change it to range from 0 to 100.

Reply:
Actually thinking about it....it would make sense for the range to be 0 to 1. The standard deviation of points on a terrain shouldn't have a standard deviation of over 1. The only reason why its being increased is because I am applying a multiplier. So at this point I think [0,1] is the best interval and then we will tune the risk_multiplier accordingly.

Reply:
Ok then, I'll change the const value to 1 for now then